versions.tf

```
@@ -1,3 +1,10 @@
 terraform {
-  required_version = ">= 0.12.14"
+  required_version = ">= 0.12.28"
```

```suggestion
  required_version = ">= 0.12.19"
```
Don't feel strongly on this.. if there's a reason for newer, go for it.

---

`0.12.28` is when they added support for the new way of specifying providers that includes `source`